### PR TITLE
Add JavaScript snippets

### PR DESCRIPTION
**Pull request by Axorax**
# Description

## Type of Change

- [x] ✨ New snippet
- [ ] 🛠 Improvement to an existing snippet
- [ ] 🐞 Bug fix
- [ ] 📖 Documentation update
- [ ] 🔧 Other (please describe):

## Checklist

- [x] I have tested my code and verified it works as expected.
- [x] My code follows the style and contribution guidelines of this project.
- [x] Comments are added where necessary for clarity.
- [x] Documentation has been updated (if applicable).
- [x] There are no new warnings or errors from my changes.

## Related Issues

Closes #

## Screenshots (Optional)

<details> 
<summary>Click to view screenshots</summary> 

 

</details>

### CODE REVIEW

File: public/data/java.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            "author": "axorax"
+          },
+          {
+            "title": "Count Vowels",
+            "description": "Counts the number of vowels in a string.",
+            "code": [
+              "public class StringManipulation {",
+              "    public static int countVowels(String s) {",
+              "        int count = 0;",
+              "        String vowels = 'aeiouAEIOU';",
```
[the remark this answers is not in the document]
Comment (Axorax) [reply public/data/java.json:70]: ohh mb

**Comment by Axorax** [comment]
wait i forgot something

**Comment by Axorax** [comment]
should be good now

**Review by Mathys-Gasnier — APPROVED** [review]
Looks good, some really usefull stuff in there

**Pull request merged**